lib/server/worker.js

```diff
@@ -52,6 +52,16 @@ exports.create = function() {
     }
   });
 
+  server.route({
+    method: 'DELETE',
+    path: '/a/{id}',
+    config: {
+      handler: function delete(req, reply) {
+        img.delete(req.params.id).done(reply, reply);
```

Comment:
@pdehaan, authentication was already handled [here](https://github.com/mozilla/fxa-profile-server/pull/90/files#diff-0ae6eb352cb79b1df84aa948c95de70dR36), but it's a good point that if that's every dropped for any reason, this would be problematic. 

Tests to the rescue.